grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/migrate/android/LintConfigData.kt

```diff
@@ -27,6 +27,18 @@ data class LintConfigData(
     val baselinePath: String? = null,
     val lintChecks: List<BazelDependency>? = null
 ) {
+
+    private fun lintChecksString(): String? {
```

Comment:
Please consider updating Grazel to use this so we have an example of how this works.